pkg/quota/registry.go

```diff
@@ -7,14 +7,16 @@ import (
 	"github.com/openshift/origin/pkg/quota/image"
 )
 
-// NewRegistry returns a registry object that knows how to evaluate quota usage of OpenShift resources.
-// Registry for resource quota controller cannot be used with resource quota admission plugin and
-// vice versa. If the registry will be used in admission plugin, pass true to forAdmission.
-// See a package documentation of pkg/quota/image for more details.
-func NewRegistry(osClient osclient.Interface, forAdmission bool) kquota.Registry {
-	if forAdmission {
-		return image.NewImageRegistryForAdmission(osClient)
-	} else {
-		return image.NewImageRegistry(osClient)
-	}
+// NewOriginQuotaRegistry returns a registry object that knows how to evaluate quota usage of OpenShift
```

Comment:
Or do we want to keep it so we can add other things to the registry as we quota more than just image related bits?